core/src/main/java/org/apache/brooklyn/core/objs/proxy/InternalPolicyFactory.java

```diff
@@ -125,7 +124,6 @@ public InternalPolicyFactory(ManagementContextInternal managementContext) {
             for (Map.Entry<ConfigKey<?>, Object> entry : spec.getConfig().entrySet()) {
                 pol.config().set((ConfigKey)entry.getKey(), entry.getValue());
             }
-            ConfigConstraints.assertValid(pol);
```

Comment:
Because of unresolvable config that needs to know the entity context, which is a valid thing.